core/src/main/kotlin/no/strooped/view/screen/JoinGameScreen.kt

```diff
@@ -21,9 +21,9 @@ import no.strooped.TextureSizes
 /**
  * Uses https://otter.tech/an-mvc-guide-for-libgdx/ as inspiration
  * */
-    const val FONT_SIZE = 5.0f
-    const val NAME_OF_SKIN = "background"
-    const val TEMP_SIZE = 10
+val FONT_SIZE_INPUT_FIELDS = TextureSizes.adjustedFontSize(2.0f)
+const val NAME_OF_SKIN = "background"
+const val TEMP_SIZE = 10
```

Comment:
Try to avoid using «temp» in a global or function scope, as it doesn’t easily portray any meaning to the reader

Reply:
Okay, I'll try and rename it

Reply:
Done